code/game/mecha/combat/phazon.dm

```diff
@@ -342,6 +342,88 @@
 	ME.attach(src)
 	max_ammo()
 
+///Legion balloon
+
+/obj/mecha/combat/phazon/vertibird/ballon
+	name = "\improper Legion Recon ballon"
```

Comment:
```suggestion
	name = "\improper Legion Recon balloon"
```

Reply:
All changed

Reply:
> All changed

Hey! The type paths above are still listed under `vertibird/ncr` instead of `vertibird/balloon` so it won't work as intended

Reply:
Thanks again Ron for the fixes